variables.tf

```diff
@@ -97,7 +97,7 @@ variable "ca_cert_identifier" {
 
 variable "performance_insights_enabled" {
   type        = bool
-  description = "Enable performance insights for RDS?"
+  description = "Enable performance insights for RDS? This is disabled by default. However a user can enable it if certain insights are needed from the Database. However, the user should ensure it is disabled once the desired outcome is achieved."
```

Comment:
"Enable performance insights for RDS? Note: the user should ensure insights are disabled once the desired outcome is achieved."
Comment

I don't think you need to explicitly mention the other things as that's implicit from reading the code

Reply:
Thanks Jazz. Updated it.